packages/combo-box/test/selecting-items.test.js

```diff
@@ -83,14 +83,15 @@ describe('selecting items', () => {
   it('should close the dropdown when reselecting the current value', () => {
     comboBox.value = 'foo';
     comboBox.open();
-    selectItem(comboBox, 0);
+    getFirstItem(comboBox).click();
```

Comment:
Could we use `clickItem(comboBox, 0)` here?

Reply:
Done. Updated a few other tests as well for consistency.